base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -1,6 +1,24 @@
 <?xml version='1.0' encoding='UTF-8' standalone='yes'?>
 <descriptions>
   <opis>
+	<entry>
+      <key>Aeroflex</key>
+      <value>
+        <type>FUNCTION_GENERATOR</type>
+        <path>aeroflex/aeroflex.opi</path>
```

Comment:
If the python script is removed the OPI file can probably be taken out from the folder.

Reply:
- I don't see the harm in keeping the 'Settings' grouping. It adds a symmetry that I think looks better than not having it
- I've no strong feelings about keeping `Actual` but it is in the LabView GUI
- I removed the units from the Carrier Frequency readout as the dropdown tells the user what units the reading is in